Iss2 create review form

tynanook opened the pull request:
@yuens1002 @JBigs206 
This is the branch for iss2, creating a review form. [iss2-create-review-form](https://github.com/suwebdev/proud-dew/issues/2).

This form is accessed from the poem.html page (iss1), from the add review button. Once the form is submitted, the page should return to the poem page. It is static right now; as task must be added to make form submission updated the database and route to the updated poem page.

Commit:
added empty poem.html page

Commit:
created empty review.html page

tynanook committed:
added poem.html page. made modification to pages.html to show functionality. Poem.html has reviews listed at bottom of page, as well as button to add new reviews. this must work with review form for functionality, which is iss2.

tynanook committed:
created poem review form. this form is designed to work with the 'poem.html' page designed in iss1-create-poem-page, which iss1 added a poem.html page which included a button to add a review.

tynanook committed:
removed poems.html from my file as it is not part of my branch.

tynanook committed:
removed blank spaces in poem.html

tynanook commented:
I added @yuens1002 and @JBigs206  as potential reviewers as they are likely to review iss1, the poem.html page, which has the list of reviews.

tynanook commented:
@yuens1002 @JBigs206 @wibblywink, upon receiving comment and any necessary updates/fixes by me, this branch is ready for merging.

tynanook commented:
@yuens1002 
FYI, I removed your pages.html from this branch. I had modified the pages.html to test the functionality of the page.html and the review form. I removed this based on your slack message that files were modified that you didn't expect. Let me know if you think I should have kept the modified version of the pages.html so reviewers could have more easily testing my page.html in this branch.

JBigs206 commented on poem-review-form.html:
```diff
@@ -0,0 +1,185 @@
+
+
+
+ 
```
We should remove the empty lines from 127 - 150 and have the closing div and proceeding code on line 151 moved up to line 127.

yuens1002 commented:
are we allowing anyone to leave comment or just a user who's logged in?  this form may be better off at the bottom of each poem page b/c it is not a standalone action.  the action is related to the poem the user is about leave a comment.  i'm also not seeing styles and files which you used for the page.  it would be nice for just previewing your page in the context of the entire project.

ghost commented:
@yuens1002 I think reviews should be for people logged in, so I agree with the idea to include it on the  poem page rather than have a standalone action.

JBigs206 commented:
@yuens1002 I'm able to preview the page from the Github desktop if you go to the branch "iss1_iss2_iss3_poem-page_review_rating" then open it in "explorer" option under settings. The individual branches don't have styles currently but the issue 1, 2, and 3 branch seems to be showing all the styles.

JBigs206 commented:
@yuens1002 @wibblywink I also agree that user should be logged in before they can comment or review.

tynanook committed:
fetch/merge'd from master

tynanook committed:
added poem.html, same as before w/no changes, just added to fetch'd master

tynanook committed:
fetched, updating branch files with master. 're'-added the poem-review-form.html. Also included in this branch of iss2 is the poem.html file from iss1 so that the review form can be viewed by a peer reviewer. Therefore, review, approve iss1 before iss2.

tynanook committed:
Merge branch 'iss2-create-review-form' of github.com:suwebdev/proud-dew into iss2-create-review-form
Merging branches to bring origin uptodate`

tynanook committed:
removed blank lines in poem.html

tynanook committed:
removed comment from poem.html

tynanook commented:
@yuens1002 @wibblywink @JBigs206
I added the 'poem.html' page (created in iss1 branch) to allow viewing of the review form (it includes the review button). Because of this, this branch will not be merged until iss1 is approved and merged.

_The following is from iss1, but applies here, to get to the review button:_
I merged the master pages into this branch to allow easier viewing of the 'poem.html' page. 
I modified the 'poems.html' to allow functional viewing of the 'poem.html' page. To view the 'poem.html' page: from the homepage, click on the 'poems.html' --> 'titles' in the navbar menu, then either click on 'Aaron' or 'Abacus'.

yuens1002 commented:
1. same problems in the code in poem.html from iss1
2. include side nav

general house keeping.  pls delete branches that aren't intended for pull requests.
